storage/remote/read.go

```diff
@@ -42,16 +43,27 @@ var remoteReadQueriesTotal = prometheus.NewCounterVec(
 	[]string{remoteName, endpoint},
 )
 
+var remoteReadQueriesHistogram = prometheus.NewHistogramVec(
+	prometheus.HistogramOpts{
+		Namespace: namespace,
+		Subsystem: subsystem,
+		Name:      "remote_read_request_duration_seconds",
+		Help:      "Histogram of the latency for remote read requests.",
+		Buckets:   prometheus.DefBuckets,
```

Comment:
Since the default remote read timeout is 60s I think it would be worth appending 25s and 60s as additional buckets on top of the default.

Reply:
@njingco you can look at this PR as an example of how to add additional buckets: https://github.com/prometheus/prometheus/pull/7323

Reply:
yeah for sure!